applications/rag/README.md

```diff
@@ -24,15 +24,25 @@ Preinstall the following on your computer:
 ### Infra Setup
 
 #### GKE Cluster Setup
-1. This sample assumes a GKE cluster already exists. If not, please follow the [instructions](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/infrastructure/README.md) to create a GKE cluster.
 
-TODO: Add GKE cluster requirements for a successful installation.
+Set your cluster name and cloud region:
+```
+CLUSTER_NAME=rag-cluster
+CLUSTER_REGION=us-central1
+```
+
+1. This sample assumes a GKE cluster already exists. If not, we recommend creating an Autopilot cluster on the latest available version:
+```
+gcloud container clusters create-auto $CLUSTER_NAME --location $CLUSTER_REGION --cluster-version 1.29.1-gke.1589000
+```
+
+To create a GKE Standard cluster, please refer to our [LLM on GKE guide](https://cloud.google.com/kubernetes-engine/docs/tutorials/serve-multiple-gpu#create_cluster_node_pool) for cluster configuration parameters.
```

Comment:
I'm assuming you tested it works E2E /pods come up with this create command for standard

Reply:
Seems this was confusing. I only meant it as a guide for setting up a cluster with GPUs. Removed and split out the Autopilot section, as you recommended.